From c82457a822c42333a2776ab8a8fdb4f2304ede0c Mon Sep 17 00:00:00 2001
From: Christopher Homberger <christopher.homberger@web.de>
Date: Tue, 4 Aug 2026 17:30:38 +0200
Subject: [PATCH 04/79] fix: remove unused hr variable

Ported to the CrossOver 26.3 tree from 0003-fix-remove-unused-hr-variable.patch.
---
 dlls/windows.devices.enumeration/information.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dlls/windows.devices.enumeration/information.c b/dlls/windows.devices.enumeration/information.c
index 5b41c44..46db8b6 100644
--- a/dlls/windows.devices.enumeration/information.c
+++ b/dlls/windows.devices.enumeration/information.c
@@ -356,8 +356,6 @@ HRESULT device_information_create( const DEV_OBJECT *obj, IDeviceInformation **i
 
 HRESULT async( IUnknown *invoker, IUnknown *param, PROPVARIANT *result, BOOL called_async )
 {
-    HRESULT hr;
-
     TRACE( "invoker %p, param %p, result %p\n", invoker, param, result );
 
     result->vt = VT_UNKNOWN;
